atomic_reactor/plugins/cachi2_init.py

```diff
@@ -135,6 +138,15 @@ def process_remote_sources(self) -> List[Dict[str, Any]]:
                 remote_source_data["ref"]
             )
 
+            if has_git_submodule_manager(remote_source_data):
+                update_submodules(source_path_app)
+                repo = git.Repo(str(source_path_app))
+                git_submodules = {
+                    "sbom": get_submodules_sbom(repo),
+                    "request_json": get_submodules_request_json_deps(repo)
```

Comment:
Naming nitpick: these two are just the `.components` part of the SBOM and the `.dependencies` part of the request JSON. But the code that uses these is simple enough that the confusion is not too large, so renaming is optional

Reply:
I updated more parts, trying to be more consistent with mentioning components